kotlinpoet/src/main/java/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -213,7 +213,11 @@ internal class CodeWriter constructor(
 
   fun emitCode(format: String, vararg args: Any?) = emitCode(CodeBlock.of(format, *args))
 
-  fun emitCode(codeBlock: CodeBlock, isConstantContext: Boolean = false) = apply {
+  fun emitCode(
+    codeBlock: CodeBlock,
+    isConstantContext: Boolean = false,
+    ensureTrailingNewline: Boolean = false
```

Comment:
Both call sites pass `true`, should we just make it the default? Or even remove this parameter, would there be cases where we don't want to ensure the new line?

Reply:
Both that specify it do, but not every call site necessarily needs it. Was trying to be least invasive as possible starting out

![image](https://user-images.githubusercontent.com/1361086/62815545-2eb05d80-bae8-11e9-9e6e-c3bd0f91eb7b.png)

Reply:
Ah, makes sense!